docs/images/20220204_missionlz_as_of_Feb2022_light.svg

```diff
@@ -2468,8 +2468,8 @@
 							x="-3.23" dy="1.2em" class="st9">Initiatives</tspan></text>			</g>
 		</g>
 		<g id="group1360-746" transform="translate(422.293,-524.764)" v:mID="1360" v:groupContext="group">
-			<title>Security Center.1214</title>
-			<desc>Azure Security Center / Defender</desc>
+			<title>Microsoft Defender.1214</title>
+			<desc>Microsoft Defender for Cloud</desc>
```

Comment:
Up to you if you want to make this change (we can instead push up a similar change in the source Visio in a future branch), but, L2524-2525 I think might update the green icon's label too:

https://github.com/Azure/missionlz/blob/62fa4318de3789eff2ab893135f7da67464781f8/docs/images/20220204_missionlz_as_of_Feb2022_light.svg?short_path=70556b0#L2524-L2525

Something like:

```svg
				<text x="-15.93" y="619.2" class="st8" v:langID="1033"><v:tabList/>Microsoft Defender<tspan
							x="0" dy="1.2em" class="st9">For Cloud</tspan></text>			</g>
```

Should generate:

![image](https://user-images.githubusercontent.com/4622125/155383788-217cead9-a2dc-4df9-870f-7595377ce8a8.png)

Reply:
Fixed!